assets/javascripts/discourse/templates/components/campaign-banner.hbs

```diff
@@ -0,0 +1,82 @@
+{{#if shouldShow}}
+  {{d-button
+    icon="times"
+    action="dismissBanner"
+    class="close"
+  }}
+  <div class="campaign-banner-info">
+    <h2 class="campaign-banner-info-header">{{i18n 'discourse_subscriptions.campaign.title'}}</h2>
+      <p class="campaign-banner-info-description">{{i18n 'discourse_subscriptions.campaign.body'}}</p>
+      {{#if product}}
+        {{#link-to "s.show" product disabled=product.subscribed class="btn btn-primary campaign-banner-info-button"}}
+          {{d-icon "far-heart"}} {{d-icon "heart" class="hover-heart"}} {{i18n 'discourse_subscriptions.campaign.button'}}
+        {{/link-to}}
+      {{else}}
+        {{#link-to "s" class="btn btn-primary campaign-banner-info-button"}}
+          {{d-icon "far-heart"}} {{d-icon "heart" class="hover-heart"}} {{i18n 'discourse_subscriptions.campaign.button'}}
+        {{/link-to}}
+      {{/if}}
+  </div>
+  <div class="campaign-banner-progress">
+    {{#if isGoalMet}}
+      <p class="campaign-banner-progress-success">{{i18n 'discourse_subscriptions.campaign.goal'}}!</p>
+      {{#if subscriberGoal}}
+        <p class="campaign-banner-progress-description">
+          <strong>{{subscribers}}</strong> {{i18n 'discourse_subscriptions.campaign.of'}}
```

Comment:
For translation purposes you should create one key that accepts `subscribers` and `goalTarget` as variables rather than split up with `of`. Not all languages would work with it split up like this.